index.html

```diff
@@ -4,15 +4,46 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
+<header>
+    <div class="top-logo">
+        <a href="#">Abbas</a>
     </div>
+    <nav class="Navbar">
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+</header>
+    <main>
+        <div class="informative-top">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
+            </div>
+            <div class="heading-description">
+                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+            </div>
+            <div class="touch-links">
+                <h3><a href="#">LET'S CONNECT</a></h3>
+                <div class="social-bar">
+                    <a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a>
```

Comment:
- [ ] As mentioned by the previous reviewer, you should use the `<section>` tag because is serves as a generic sectioning element and ultimately help assistive technology make sense of your HTML document. It appears you still want to use the `main` tag, no problem with that. I strongly suggest you should change the `<div class="informative-top">` to a section tag. Essentially, this headline section must be wrapped in a `<section>` tag. This is a core requirement for this project. Kindly consider making these changes.